dietpi/dietpi-software

```diff
@@ -8966,6 +9010,43 @@ _EOF_
 			pihole -w freegeoip.app --comment 'Used by dietpi-banner and dietpi-vpn to obtain your public IP and location'
 
 		fi
+		
+		software_id=171 # AdGuardHome
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+			
+			if [[ ! -f '/opt/AdGuardHome/AdGuardHome.yaml' ]]; then
+				dps_index=$software_id Download_Install 'AdGuardHome.yaml' /opt/AdGuardHome/AdGuardHome.yaml
+				G_CONFIG_INJECT 'password:[[:blank:]]' "  password: $(htpasswd -bnBC 10 "" $GLOBAL_PW | tr -d ':\n' | sed 's/$2y/$2a/')" /opt/AdGuardHome/AdGuardHome.yaml
+			fi
+
+			# Service
+			cat << _EOF_ > /etc/systemd/system/AdGuardHome.service
+[Unit]
+Description=AdGuard Home: Network-level blocker
+ConditionFileIsExecutable=/opt/AdGuardHome/AdGuardHome
+After=syslog.target network-online.target
+
+[Service]
+StartLimitInterval=5
+StartLimitBurst=10
```

Comment:
These two need to go to the `[Unit]` section and `StartLimitBurst` is now called `StartLimitIntervalSec`: https://www.freedesktop.org/software/systemd/man/systemd.unit.html#StartLimitIntervalSec=interval

Reply:
for the initial setup, I used the original AGH service definition. Seems they are not up-to-date

Reply:
done